nca/Utils/CmdlineRunner.py

```diff
@@ -9,23 +9,33 @@
 
 import subprocess
 import os
+import sys
 from fnmatch import fnmatch
 
 
 class CmdlineRunner:
     """
     A stateless class with only static functions to easily get k8s and calico resources using kubectl and calicoctl
     """
+    # a static variable to indicate if we want to ignore errors from running executable command
+    ignore_live_cluster_err = False
+
     @staticmethod
-    def run_and_get_output(cmdline_list):
+    def run_and_get_output(cmdline_list, helm_flag=False):
```

Comment:
I think a better approach would be to set a k8s live cluster flag (default as False, to be set true when called from the relevant functions). Since `run_and_get_output` might be extended for other use cases in the future.